src/OpenTracing/Standard/Format.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace OpenTracing\Standard;
+
+class Format {
```

Comment:
we will definitely want comments about the appropriate carrier for each format at some point